ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
So I'm still a bit confused about your confusion. Let me try to break down how I would go about solving this, maybe it will help:

So if you look up this:

https://treeherder.mozilla.org/api/project/mozilla-inbound/performance/signatures/?signature=32206df28ab3b52e15d26f73178ee78e2af7e760

You get this response:

```
{
    "32206df28ab3b52e15d26f73178ee78e2af7e760": {
        "suite": "ts_paint",
        "subtest_signatures": [
            "726a741fd1dd1e9fb0b85bc9733658f6eabd07df"
        ],
        "option_collection_hash": "102210fe594ee9b33d82058545b1ed14f4c8206e",
        "machine_platform": "linux64"
    }
}
```

You see that the signature id is right there in the dictionary. You can just look up the properties based on the signature hash and then create the name of the series from that. To get all the signatures, just do a bigger query:

https://treeherder.mozilla.org/api/project/mozilla-inbound/performance/signatures/?signature=<signature hash1>&signature=<signature hash 2>...

Then write a routine to unpack the values and associate them with particular series.